src/prefect/settings/models/server/api.py

```diff
@@ -31,6 +31,11 @@ class ServerAPISettings(PrefectBaseSettings):
         description="The API's port address (defaults to `4200`).",
     )
 
+    base_path: Optional[str] = Field(
+        default=None,
+        description="The base URL path to serve the API under, e.g., /v2/api",
+    )
```

Comment:
the `examples` kwarg can be used to provide a list of possible values for the field